pylearn2/sandbox/nlp/models/tests/test_mlp.py

```diff
@@ -0,0 +1,15 @@
+'''
+Created on Jan 17, 2015
+
+@author: Minh Ngoc Le
```

Comment:
Whichever you prefer. No official responsibilities come with putting your name at the top; it's mostly for credit I guess (because for any questions we can find the one who wrote the code anyway through Git blame).